css/style.css

```diff
@@ -1,9 +1,20 @@
-
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
 body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  margin: 0;
+  box-sizing: border-box;
+  padding: 0%;
+}
+h2 {
+  font-weight: lighter;
+}
+header {
```

Comment:
Cosmetics: It's a good practice to leave an empty line after each css rule you write:
```
h2 {
  ...
}

header {
  ...
}
```